rust/cymbal/src/types/db.rs

```diff
@@ -0,0 +1,84 @@
+use uuid::Uuid;
+
+pub struct ErrorTrackingIssueFingerprint {
+    pub id: Uuid,
```

Comment:
This is a `class ErrorTrackingIssueFingerprint(models.Model)`, meaning the ID type is i32... I think this was straightforwardly a mistake on our part (I'm pretty sure I stamped the review), and I guess we should ship some migrations to fix it 🙃

Reply:
I think we _need_ it to be something we won't run out of, and I don't think an i32 is large enough to be confident of that. My general stance is that a UUID is the default, and anything else should only be used if strictly necessary.